### PR TITLE
PM 2.0 GA (Final) Dev Review

### PR DESCRIPTION
**Pull request by mcherem**
Final technical review by Dev.

### CODE REVIEW

File: modules/ROOT/pages/setup.adoc [hunk 1]
```diff
@@ -113,10 +114,23 @@ The follow the steps explain how to setup the Payload Storage API:
 . Download the pre-packaged API specification (in RAML) and sample implementation template from Anypoint Exchange.
 Install this API on a Mule runtime that can access the storage solution.**[MISSING INFO - link from Anypoint Exchange for the pre-packaged API and integration app from Manik / Pablo]**
```
Comment (pmantellini) [inline modules/ROOT/pages/setup.adoc]: This is not clear for me, I would just remove the the whole "install..." phrase and add the Exchange link in the first one.

File: modules/ROOT/pages/setup.adoc [hunk 2]
```diff
@@ -113,10 +114,23 @@ The follow the steps explain how to setup the Payload Storage API:
...
+** Certificate (.PEM format) and connection type
+
+[xref] to HTTPS Send Configuration for complete details?
+
 You'll need these details while associating Anypoint Partner Manager to your Payload Storage API. +
 
-NOTE: Your payload storage API must be accessible over the internet via HTTPS. Ensure that the appropriate authentication mechanisms have been implemented to make this happen.
+NOTE: Your payload storage API must be accessible over the internet via HTTPS.
+Ensure that the appropriate authentication mechanisms have been implemented to make this happen.
+For example, if you are using AS2, you'll need to understand from your administrator what the hostname is so you can tell your partner where to send the AS2 transmission.
```
Comment (pmantellini) [inline modules/ROOT/pages/setup.adoc]: This doesn't look related to the previous statement

File: modules/ROOT/pages/setup.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: Fixed.

File: modules/ROOT/pages/setup.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/setup.adoc]: REmoved per Pablo's comments on B2BSOULT-2387

**Review by amitfernandes — APPROVED** [review]
approved
